src/storage/src/error.rs

```diff
@@ -539,9 +540,10 @@ impl ErrorExt for Error {
             ConvertChunk { source, .. } => source.status_code(),
             MarkWalObsolete { source, .. } => source.status_code(),
             DecodeParquetTimeRange { .. } => StatusCode::Unexpected,
-            RateLimited { .. } | StopScheduler { .. } | CompactTaskCancel { .. } => {
-                StatusCode::Internal
-            }
+            RateLimited { .. }
+            | StopScheduler { .. }
+            | CompactTaskCancel { .. }
+            | DuplicateFlush { .. } => StatusCode::Internal,
```

Comment:
`DuplicateFlush` prefer to return `InvalidArguments`.

Reply:
This is an unexpected issue, so I use code internal. Maybe `StatusCode::Unexpected` is better.